<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta name="google-site-verification" content=""> <meta name="msvalidate.01" content=""> <meta http-equiv="Permissions-Policy" content="interest-cohort=()"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>publications | PLN-team</title> <meta name="author" content="Poisson Lognormal Models"> <meta name="description" content="Publications in academic journals and conferences by categories in reversed chronological order"> <meta name="keywords" content="Poisson Lognormal, Multivariate Statistics, Latent variable Models, Count data, Variational Inference"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?19f3075a2d19613090fe9e16b564e1fe" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/PLNmodels.png?68946a1f9a3c9d200231229459aa6884"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://pln-team.github.io/publications/"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?e74e74bf055e5729d44a7d031a5ca6a5" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?96d6b3e1c3604aca8b6134c7afdd5db6"></script> <script src="/assets/js/dark_mode.js?9b17307bb950ffa2e34be0227f53558f"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/">PLN-team</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories</a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">people</a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item" href="/main_contributors/">main contributors</a> <div class="dropdown-divider"></div> <a class="dropdown-item" href="/phd/">PhD Students</a> <div class="dropdown-divider"></div> <a class="dropdown-item" href="/msc/">MsC Students</a> <div class="dropdown-divider"></div> <a class="dropdown-item" href="/others/">Others contributors</a> </div> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">support</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">Publications in academic journals and conferences by categories in reversed chronological order</p> </header> <article> <h2 id="methodological-publications">Methodological publications</h2> <div class="publications"> <h2 class="bibliography">2021</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Front Ecol Evol</abbr></div> <div id="PLNfrontiers" class="col-sm-8"> <div class="title">The Poisson-Lognormal Model as a Versatile Framework for the Joint Analysis of Species Abundances</div> <div class="author"> Julien Chiquet, <a href="https://mahendra-mariadassou.github.io/" rel="external nofollow noopener" target="_blank">Mahendra Mariadassou</a>, and <a href="https://scj-robin.github.io/" rel="external nofollow noopener" target="_blank">Stéphane Robin</a> </div> <div class="periodical"> <em>Frontiers in Ecology and Evolution</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.frontiersin.org/articles/10.3389/fevo.2021.588292/pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="abstract hidden"> <p>Joint Species Distribution Models (JSDM) provide a general multivariate framework to study the joint abundances of all species from a community. JSDM account for both structuring factors (environmental characteristics or gradients, such as habitat type or nutrient availability) and potential interactions between the species (competition, mutualism, parasitism, etc.), which is instrumental in disentangling meaningful ecological interactions from mere statistical associations. Modeling the dependency between the species is challenging because of the count-valued nature of abundance data and most JSDM rely on Gaussian latent layer to encode the dependencies between species in a covariance matrix. The multivariate Poisson-lognormal (PLN) model is one such model, which can be viewed as a multivariate mixed Poisson regression model. Inferring such models raises both statistical and computational issues, many of which were solved in recent contributions using variational techniques and convex optimization tools. The PLN model turns out to be a versatile framework, within which a variety of analyses can be performed, including multivariate sample comparison, clustering of sites or samples, dimension reduction (ordination) for visualization purposes, or inferring interaction networks. This paper presents the general PLN framework and illustrates its use on a series a typical experimental datasets. All the models and methods are implemented in the <monospace>R</monospace> package <monospace>PLNmodels</monospace>, available from <ext-link ext-link-type="uri" xlink:href="http://cran.r-project.org" xmlns:xlink="http://www.w3.org/1999/xlink">cran.r-project.org</ext-link>.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">PLNfrontiers</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Chiquet, Julien and Mariadassou, Mahendra and Robin, Stéphane}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{The Poisson-Lognormal Model as a Versatile Framework
                    for the Joint Analysis of Species Abundances}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Frontiers in Ecology and Evolution}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{188}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fevo.2021.588292}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2296-701X}</span><span class="p">,</span>
  <span class="na">biorxiv</span> <span class="p">=</span> <span class="s">{10.1101/2020.10.07.329383v1}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">ICML</abbr></div> <div id="2019_ICML_CMR" class="col-sm-8"> <div class="title">Variational Inference for sparse network reconstruction from count data</div> <div class="author"> Julien Chiquet, Stephane Robin, and <a href="https://mahendra-mariadassou.github.io/" rel="external nofollow noopener" target="_blank">Mahendra Mariadassou</a> </div> <div class="periodical"> <em>In Proceedings of the 36th International Conference on Machine Learning</em>, 09–15 jun 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://arxiv.org/abs/1806.03120" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">arXiv</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="http://proceedings.mlr.press/v97/chiquet19a/chiquet19a.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="abstract hidden"> <p>The problem of network reconstruction from continuous data has been extensively studied and most state of the art methods rely on variants of Gaussian Graphical Models (GGM). GGM are unfortunately badly suited to sparse count data spanning several orders of magnitude. Most inference methods for count data (SparCC, REBACCA, SPIEC-EASI, gCoda, etc) first transform counts to pseudo-Gaussian observations before using GGM. We rely instead on a PoissonLogNormal (PLN) model where counts follow Poisson distributions with parameters sampled from a latent multivariate Gaussian variable, and infer the network in the latent space using a variational inference procedure. This model allows us to (i) control for confounding covariates and differences in sampling efforts and (ii) integrate data sets from different origins. It is also competitive in terms of speed and accuracy with state of the art methods.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">2019_ICML_CMR</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Variational Inference for sparse network
                    reconstruction from count data}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Chiquet, Julien and Robin, Stephane and Mariadassou, Mahendra}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{Proceedings of the 36th International Conference on
                    Machine Learning}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1162--1171}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span><span class="p">,</span>
  <span class="na">editor</span> <span class="p">=</span> <span class="s">{Chaudhuri, Kamalika and Salakhutdinov, Ruslan}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{97}</span><span class="p">,</span>
  <span class="na">series</span> <span class="p">=</span> <span class="s">{Proceedings of Machine Learning Research}</span><span class="p">,</span>
  <span class="na">address</span> <span class="p">=</span> <span class="s">{Long Beach, California, USA}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="s">{09--15 Jun}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{PMLR}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://proceedings.mlr.press/v97/chiquet19a.html}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Ann Appl Stat</abbr></div> <div id="2018_AOAS_cmr" class="col-sm-8"> <div class="title">Variational inference for probabilistic Poisson PCA</div> <div class="author"> Julien Chiquet, <a href="https://mahendra-mariadassou.github.io/" rel="external nofollow noopener" target="_blank">Mahendra Mariadassou</a>, and <a href="https://scj-robin.github.io/" rel="external nofollow noopener" target="_blank">Stéphane Robin</a> </div> <div class="periodical"> <em>Ann. Appl. Statist.</em>, 09–15 jun 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://arxiv.org/abs/1703.06633" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">arXiv</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://projecteuclid.org/journals/annals-of-applied-statistics/volume-12/issue-4/Variational-inference-for-probabilistic-Poisson-PCA/10.1214/18-AOAS1177.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="abstract hidden"> <p>Many application domains, such as ecology or genomics, have to deal with multivariate non-Gaussian observations. A typical example is the joint observation of the respective abundances of a set of species in a series of sites aiming to understand the covariations between these species. The Gaussian setting provides a canonical way to model such dependencies but does not apply in general. We consider here the multivariate exponential family framework for which we introduce a generic model with multivariate Gaussian latent variables. We show that approximate maximum likelihood inference can be achieved via a variational algorithm for which gradient descent easily applies. We show that this setting enables us to account for covariates and offsets. We then focus on the case of the Poisson-lognormal model in the context of community ecology. We demonstrate the efficiency of our algorithm on microbial ecology datasets. We illustrate the importance of accounting for the effects of covariates to better understand interactions between species.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">2018_AOAS_cmr</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Chiquet, Julien and Mariadassou, Mahendra and Robin, St\'ephane}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Variational inference for probabilistic Poisson PCA}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Ann. Appl. Statist.}</span><span class="p">,</span>
  <span class="na">fjournal</span> <span class="p">=</span> <span class="s">{The Annals of Applied Statistics}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{2674-2698}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1932-6157}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1214/18-AOAS1177}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> </div> <h2 id="applications-of-pln-in-ecology-microbiolgy-genomics">Applications of PLN in Ecology, Microbiolgy, Genomics</h2> <div class="publications"> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Anim. Microbiome</abbr></div> <div id="mariadassou2023microbiota" class="col-sm-8"> <div class="title">Microbiota members from body sites of dairy cows are largely shared within individual hosts throughout lactation but sharing is limited in the herd</div> <div class="author"> <a href="https://mahendra-mariadassou.github.io/" rel="external nofollow noopener" target="_blank">Mahendra Mariadassou</a>, Laurent X Nouvel, Fabienne Constant, and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'Diego P Morgavi, Lucie Rault, Sarah Barbey, Emmanuelle Helloin, Olivier Rué, Sophie Schbath, Frederic Launay, Olivier Sandra, Rachel Lefebvre, Yves Le Loir, Pierre Germon, Christine Citti, Sergine Even' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> <em>Animal Microbiome</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a href="https://animalmicrobiome.biomedcentral.com/counter/pdf/10.1186/s42523-023-00252-w.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Front. microbiol.</abbr></div> <div id="Thierry2023" class="col-sm-8"> <div class="title">Microbial communities of a variety of 75 homemade fermented vegetables</div> <div class="author"> Anne Thierry, Marie-Noelle Madec, Victoria Chuat, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Anne-Sophie Bage, Olivier Picard, Cécile Grondin, Olivier Rué, Mahendra Mariadassou, Laurent Marché, Florence Valence' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Frontiers in Microbiology</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Ecol. Lett.</abbr></div> <div id="Facon2021" class="col-sm-8"> <div class="title">Joint species distributions reveal the combined effects of host plants, abiotic factors and species competition as drivers of species abundances in fruit flies</div> <div class="author"> Benoit Facon, Abir Hafsi, Maud Charlery Masselière, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Stéphane Robin, François Massol, Maxime Dubart, Julien Chiquet, Enric Frago, Frédéric Chiroleu, Pierre-François Duyck, Virginie Ravigné' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>Ecological Letters</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>The relative importance of ecological factors and species interactions for phytophagous insect species distributions has long been a controversial issue. Using field abundances of eight sympatric Tephritid fruit flies on 21 host plants, we inferred flies’ realized niches using joint species distribution modelling and network inference, on the community as a whole and separately on three groups of host plants. These inferences were then confronted to flies’ fundamental niches estimated through laboratory-measured fitnesses on host plants. Species abundances were mainly determined by host plants followed by climatic factors, with a minor role for competition between species sharing host plants. The relative importance of these factors mildly changed when we focused on particular host plant groups. Despite overlapping fundamental niches, specialists and generalists had almost distinct realized niches, with possible competitive exclusion of generalists by specialists on Cucurbitaceae and different assembly rules: specialists were mainly influenced by their adaptation to host plants while generalist abundances varied regardless of their fundamental host use.Competing Interest StatementThe authors have declared no competing interest.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Facon2021</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Facon, Benoit and Hafsi, Abir and de la Masseli{\`e}re, Maud Charlery and Robin, St{\'e}phane and Massol, Fran{\c c}ois and Dubart, Maxime and Chiquet, Julien and Frago, Enric and Chiroleu, Fr{\'e}d{\'e}ric and Duyck, Pierre-Fran{\c
                    c}ois and Ravign{\'e}, Virginie}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Joint species distributions reveal the combined
                    effects of host plants, abiotic factors and species
                    competition as drivers of species abundances in
                    fruit flies}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1111/ele.13825}</span><span class="p">,</span>
  <span class="na">biorxiv</span> <span class="p">=</span> <span class="s">{early/2021/03/03/2020.12.07.414326}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Ecological Letters}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Microbiome</abbr></div> <div id="Lejal2021" class="col-sm-8"> <div class="title">Temporal patterns in Ixodes ricinus microbial communities: an insight into tick-borne microbe interactions</div> <div class="author"> E Lejal, J Chiquet, J Aubert, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'S Robin, A Estrada-Peña, O Rue, C Midoux, M Mariadassou, X Bailly, A Cougoul, P Gasqui, JF Cosson, K Chalvet-Monfray, M Vayssier-Taussat, T Pollet' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>Microbiome</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Background Ticks transmit pathogens of medical and veterinary importance, and represent an increasing threat for human and animal health. Important steps in assessing disease risk and developing possible new future control strategies involve identifying tick-borne microbes, their temporal dynamics and interactions.Methods Using high throughput sequencing, we studied the microbiota dynamics of Ixodes ricinus from 371 nymphs collected monthly over three consecutive years in a peri-urban forest. After adjusting a Poisson Log Normal model to our data set, the implementation of a principal component analysis as well as sparse network reconstruction and differential analysis allowed us to assess inter-annual, seasonal and monthly variability of I. ricinus microbial communities as well as their interactions.Results Around 75% of the detected sequences belonged to five genera known to be maternally inherited bacteria in arthropods and potentially circulating in ticks: Candidatus Midichloria, Rickettsia, Spiroplasma, Arsenophonus and Wolbachia. The structure of the I. ricinus microbiota was temporally variable with interannual recurrence and seemed to be mainly driven by OTUs belonging to environmental genera. The total network analysis revealed a majority of positive (partial) correlations. We identified strong relationships between OTUs belonging to Wolbachia and Arsenophonus, betraying the presence of the parasitoid wasp Ixodiphagus hookeri in ticks, and the well known arthropod symbiont Spiroplasma, previously documented to be involved in the defense against parasitoid wasp in Drosophila melanogaster. Other associations were observed between the tick symbiont Candidatus Midichloria and pathogens belonging to Rickettsia, probably Rickettsia helvetica. More specific network analysis finally suggested that the presence of pathogens belonging to genera Borrelia, Anaplasma and Rickettsia might disrupt microbial interactions in I. ricinus.Conclusions Here, we identified the I. ricinus microbiota and documented for the first time the existence and recurrence of marked temporal shifts in the tick microbial community dynamics. We statistically showed strong relationships between the presence of some pathogens and the structure of the I. ricinus non-pathogenic microbes. We interestingly detected close links between some tick symbionts and the potential presence of either pathogenic Rickettsia or a parasitoid in ticks. All these new findings might be very promising for the future development of new control strategies of ticks and tick-borne diseases.Competing Interest StatementAuthor Cedric Midoux was employed by the company Irstea. The remaining authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.I. ricinusIxodes ricinusPCAPrincipal Component AnalysisOTUOperational Taxonomic UnitTBPTick-Borne Pathogens</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Lejal2021</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Lejal, E and Chiquet, J and Aubert, J and Robin, S and Estrada-Pe{\~n}a, A and Rue, O and Midoux, C and Mariadassou, M and Bailly, X and Cougoul, A and Gasqui, P and Cosson, JF and Chalvet-Monfray, K and Vayssier-Taussat, M and Pollet, T}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Temporal patterns in Ixodes ricinus microbial
                    communities: an insight into tick-borne microbe
                    interactions}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{153}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1186/s40168-021-01051-8}</span><span class="p">,</span>
  <span class="na">biorxiv</span> <span class="p">=</span> <span class="s">{early/2020/09/26/2020.09.26.314179}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Microbiome}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Adv. Ecol. Res.</abbr></div> <div id="DUBART2021367" class="col-sm-8"> <div class="title">Chapter Nine - Coupling ecological network analysis with high-throughput sequencing-based surveys: Lessons from the next-generation biomonitoring project</div> <div class="author"> Maxime Dubart, Pascal Alonso, Didac Barroso-Bergada, and <span class="more-authors" title="click to view 26 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '26 more authors' ? 'Nathalie Becker, Kevin Bethune, David A. Bohan, Christophe Boury, Marine Cambon, Elsa Canard, Emilie Chancerel, Julien Chiquet, Patrice David, Natasha de Manincor, Sophie Donnet, Anne Duputié, Benoît Facon, Erwan Guichoux, Tâm Le Minh, Sebastián Ortiz-Martínez, Lucie Piouceau, Ambre Sacco-Martret de Préville, Manuel Plantegenest, Céline Poux, Virginie Ravigné, Stéphane Robin, Marine Trillat, Corinne Vacher, Christian Vernière, François Massol' : '26 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">26 more authors</span> </div> <div class="periodical"> <em>In The Future of Agricultural Landscapes, Part III</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Biomonitoring ecosystems is necessary in order to evaluate risks and to efficiently manage ecosystems and their associated services. Agrosystems are the target of multiple stressors that can affect many species through effects cascading along food webs. However, classic biomonitoring, focused on species diversity or indicator species, might be a poor predictor of the risk of such whole-ecosystem perturbations. Thanks to high-throughput sequencing methods, however, it might be possible to obtain sufficient information about entire ecological communities to infer the functioning of their associated interaction networks, and thus monitor more closely the risk of the collapse of entire food webs due to external stressors. In the course of the ‘next-generation biomonitoring’ project, we collectively sought to experiment with this idea of inferring ecological networks on the basis of metabarcoding information gathered on different systems. We here give an overview of issues and preliminary results associated with this endeavour and highlight the main difficulties that such next-generation biomonitoring is still facing. Going from sampling protocols up to methods for comparing inferred networks, through biomolecular, bioinformatic, and network inference, we review all steps of the process, with a view towards generality and transferability towards other systems.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@incollection</span><span class="p">{</span><span class="nl">DUBART2021367</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Chapter Nine - Coupling ecological network analysis
                    with high-throughput sequencing-based surveys:
                    Lessons from the next-generation biomonitoring
                    project}</span><span class="p">,</span>
  <span class="na">editor</span> <span class="p">=</span> <span class="s">{Bohan, David A. and Dumbrell, Alex J. and Vanbergen, Adam J.}</span><span class="p">,</span>
  <span class="na">series</span> <span class="p">=</span> <span class="s">{Advances in Ecological Research}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Academic Press}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{65}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{367-430}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">hal</span> <span class="p">=</span> <span class="s">{hal-03634351v1}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{The Future of Agricultural Landscapes, Part III}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0065-2504}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1016/bs.aecr.2021.10.007}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S0065250421000222}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dubart, Maxime and Alonso, Pascal and Barroso-Bergada, Didac and Becker, Nathalie and Bethune, Kevin and Bohan, David A. and Boury, Christophe and Cambon, Marine and Canard, Elsa and Chancerel, Emilie and Chiquet, Julien and David, Patrice and {de
                    Manincor}, Natasha and Donnet, Sophie and Duputié, Anne and Facon, Benoît and Guichoux, Erwan and {Le Minh}, Tâm and Ortiz-Martínez, Sebastián and Piouceau, Lucie and {Sacco-Martret de Préville}, Ambre and Plantegenest, Manuel and Poux, Céline and Ravigné, Virginie and Robin, Stéphane and Trillat, Marine and Vacher, Corinne and Vernière, Christian and Massol, François}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Ecological networks, Network inference,
                    Next-generation biomonitoring, High throughput
                    sequencing, Logic-based machine learning,
                    Next-generation sequencing, Microbiomes, eDNA}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> </div> <h2 id="preprints">Preprints</h2> <div class="publications"> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Champion2023" class="col-sm-8"> <div class="title">OneNet – One network to rule them all: consensus network inference from microbiome data</div> <div class="author"> Camille Champion, Raphaelle Momal, Emmanuelle Le Chatelier, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Mahendra Mariadassou, Magali Berland' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> May 2023 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Irlinger" class="col-sm-8"> <div class="title">Evidence of microbial signatures in French Protected Designation of Origin milks and cheeses</div> <div class="author"> F. Irlinger, M. Mariadassou, E. Dugat-Bony, and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'O. Rué, C.Neuvéglise, P. Renault, E. Rifa, S. Theil, V. Loux, C. Cruaud, F. Gavory, V. Barbe, R. Lasbleiz, F. Gaucheron, C. Spelle, C. Delbès' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> May 2023 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">bioRxiv</abbr></div> <div id="preprint_pauvert" class="col-sm-8"> <div class="title">Microbial association networks give relevant insights into plant pathobiomes</div> <div class="author"> Charlie Pauvert, Tania Fort, Agnès Calonnec, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Julie Faivre-d’Arcier, Emilie Chancerel, Marie Massot, Julien Chiquet, Stéphane Robin, David A. Bohan, Jessica Vallance, Corinne Vacher' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> May 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Interactions between plant pathogens and other plant-associated microorganisms regulate disease. Deciphering the networks formed by these interactions, termed pathobiomes, is crucial to disease management. Our aim was to investigate whether microbial association networks inferred from metabarcoding data give relevant insights into pathobiomes, by testing whether inferred associations contain signals of ecological interactions. We used Poisson Lognormal Models to construct microbial association networks from metabarcoding data and then investigated whether some of these associations corresponded to interactions measurable in co-cultures or known in the literature, by using grapevine (Vitis vinifera) and the fungal pathogen causing powdery mildew (Erysiphe necator) as a model system. Our model suggested that the pathogen species was associated with 23 other fungal species, forming its putative pathobiome. These associations were not known as interactions in the literature, but one of them was confirmed by our co-culture experiments. The yeast Buckleyzyma aurantiaca impeded pathogen growth and reproduction, in line with the negative association found in the microbial network. Co-cultures also supported another association involving two yeast species. Together, these findings indicate that microbial networks can provide plausible hypotheses of ecological interactions that could be used to develop microbiome-based strategies for crop protection.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2024 Poisson Lognormal Models. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Last updated: January 25, 2024. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?7b30caa5023af4af8408a472dc4e1ebb"></script> <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/assets/js/copy_code.js?07b8786bab9b4abe90d10e61f7d12ff7" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script async src="https://www.googletagmanager.com/gtag/js?id="></script> <script>function gtag(){window.dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","");</script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>